presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/login/LoginScreen.kt

```diff
@@ -1,28 +1,130 @@
+import androidx.compose.material3.Icon
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.em
+import androidx.compose.ui.unit.sp
+import com.airbnb.lottie.compose.LottieAnimation
+import com.airbnb.lottie.compose.LottieCompositionSpec
+import com.airbnb.lottie.compose.rememberLottieComposition
+import com.mashup.gabbangzip.sharedalbum.presentation.R
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.Gray0
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.Gray80
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.KakaoOnPrimaryColor
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.KakaoPrimaryColor
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.PicTypography
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.SharedAlbumTheme
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.pretendard
 
 @Composable
 fun LoginScreen(onClickLoginButton: () -> Unit) {
     Column(
-        modifier = Modifier.fillMaxSize(),
-        verticalArrangement = Arrangement.Center,
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Gray0)
+            .padding(horizontal = 16.dp),
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
+        Spacer(modifier = Modifier.weight(0.5f))
         Text(
-            modifier = Modifier.wrapContentSize(),
-            text = "로그인 화면입니다",
+            text = stringResource(id = R.string.login_intro_title),
+            fontSize = 48.sp,
+            color = Gray80,
+            fontFamily = pretendard,
+            fontWeight = FontWeight.Black,
         )
-        Button(onClick = { onClickLoginButton.invoke() }) {
-            Text(text = "로그인으로 가는 버튼")
-        }
+        Spacer(modifier = Modifier.height(20.dp))
+        Text(
+            text = stringResource(id = R.string.login_intro_subtitle),
+            fontSize = 22.sp,
+            color = Gray80,
+            fontFamily = pretendard,
+            fontWeight = FontWeight.Normal,
+            letterSpacing = (-0.02).em,
+            lineHeight = 27.sp,
+            textAlign = TextAlign.Center,
+        )
+        Spacer(modifier = Modifier.weight(0.4f))
+        PicIntroLottie()
+        Spacer(modifier = Modifier.weight(1f))
+        KakaoLoginButton(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(bottom = 36.dp),
+            onClick = { onClickLoginButton() },
```

Comment:
```suggestion
            onClick = onClickLoginButton,
```
인자로 받았던거 그대로 넘길 수 있는 친구들은 새로 람다객체 만들지 않고 그대로 넘겨주는 것이 매번 새로운 객체를 만들지 않아서 효율적이지 않을까란 생각이 들었는데.. 가능한 친구들은 요런식으로 수정해보는건 어떨까?!

Reply:
[eb83c25](https://github.com/mash-up-kr/gabbangzip-Android/pull/32/commits/eb83c25a2b42ac86da701a8d02d958658a474a5c) 반영완!